src/Launcher/Launcher.jl

```diff
@@ -679,8 +730,13 @@ function launch!(
                 println("Final Λ has not been reached, resuming calculation ...")
 
                 # load data
-                l, r        = read_lattice(cp)
-                Λ, dΛ, m, a = read_checkpoint(cp, 0.0)
+                l, r              = read_lattice(cp)
+                Λ, dΛ, m, a_inter = read_checkpoint(cp, 0.0)
+                χ                 = read_χ_all(obs, Λ; verbose = false)[2]
+                
+                # update frequency mesh
+                a = get_action_empty(symmetry, r, m; S = S)
```

Comment:
@dominikkiese here a is constructed, but from the construction doen here, a.bare is not intialized. I would suggest the following:
```suggestion
                a = deepcopy(a_inter)
```

Reply:
I dont know if deepcopy works for arbitrary types. Have you tested this?

Reply:
deepcopy just works for any type, also tested it.

Reply:
Okay great, sop this should fix it then I guess.